### PR TITLE
ported to beta-3.  adding a web map as a starting point, instead of a basemap

### PR DESCRIPTION
**Pull request by zinfin**
@doneill ready for your review.

### CODE REVIEW

File: maps-app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <string name="no_maps">No maps</string>
 
     <!-- webmap urls -->
-    <string name="default_basemap_url">http://www.arcgis.com/home/item.html?id=d5e02a0c1f2b4ec399823fdd3c2fdebd</string>
+    <string name="default_basemap_url">http://www.arcgis.com/home/item.html?id=26888b0c21a44eb1ba2f26d1eb7981fe</string>
```
Comment (doneill) [inline maps-app/src/main/res/values/strings.xml]: @zinfin Is this the intended web map you want to start with? Or just a test until we have created one?

**Comment by doneill** [comment]
@zinfin Updates look good, just a question about the web map update.  My thoughts are we should hold off on updating until we decided which web map we want to use as the opening map.

**Comment by zinfin** [comment]
@doneill The webmap is just a starting point until we decide what is appropriate.  Shall I pull it out for now?

**Comment by doneill** [comment]
@zinfin I think so as it does not serve any value, especially in refactoring the current app to Quartz.  We can make the switch once we agree on a web map to start with.

**Comment by zinfin** [comment]
@doneill Removed it.  
Also, noticed some different behavior around the soft keyboard.  When I try and sign in, after entering the portal url, the app doesn't always respond to the "CONTINUE" button.  Are you currently troubleshooting this or do you want me to?

**Comment by doneill** [comment]
@zinfin You can troubleshoot that as I have not seen that as of yet.  I will merge this now.

**Pull request merged**